site/docs/master/contributions/minio.md

```diff
@@ -229,6 +229,22 @@ You must also get the Minio URL, which you can then specify as the value of the
 
 1.  Edit your `BackupStorageLocation` YAML, adding `publicUrl: <URL_FROM_PREVIOUS_STEP>` as a field under `spec.config`. You must include the `http://` or `https://` prefix.
 
+## Accessing logs with an HTTPS endpoint
+
+If you're using Minio with HTTPS, you may see unintelligible text in the output of `velero describe`, or `velero logs` commands.
+
+In order to fix this, you can add a public URL to the `BackupStorageLocation`.
+
+In a terminal, run the following:
+
+```shell
+kubectl edit backupstoragelocation default -n velero
```

Comment:
I've started trying to use `kubectl patch` commands for things like this where I can since they don't require interactive editing -- this one would be:

```
kubectl patch -n velero backupstoragelocation default --type merge -p '{"spec":{"config":{"publicUrl":"<...the value>"}}}'
```